src/sections/Brand/index.js

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { Link } from "gatsby"
```

Comment:
Also, please fix these warnings. This one and 2-3 below it, only on this file.
The warning for unused variables.